Makefile

```diff
@@ -37,10 +37,10 @@ endif
 ifndef HAS_DEP
 	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
 endif
-	dep ensure
+	cd $(DEST) && dep ensure
```

Comment:
I'm not sure I follow, is your intent to be able to run make from outside of the repos working directory?  The eval of PWD, BASE_DIR an finally DEST should tie all this together; but even so we should expect to be in the correct directory when make is called no?